flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataview/PerWindowStateDataViewStore.java

```diff
@@ -66,10 +66,7 @@ public PerWindowStateDataViewStore(
 		checkNotNull(keyedStateBackend.getKeySerializer(),
```

Comment:
This method is no longer needed and the javadoc doesn't match this method implementation. 
We can use `keyedStateBackend.getOrCreateKeyedState(windowSerializer, stateDescriptor);` instead of calling this method.